header/header.go

```diff
@@ -47,7 +47,11 @@ func MakeExtendedHeader(
 ) (*ExtendedHeader, error) {
 	var dah DataAvailabilityHeader
 	if len(b.Txs) > 0 {
-		namespacedShares, _ := b.Data.ComputeShares()
+		// TODO(@Bidon15): Do we need curLen(int)?
+		namespacedShares, _, err := b.Data.ComputeShares(uint64(0))
```

Comment:
>  If a square size
> of 0 is passed, then it is determined based on how many shares are needed to
> fill the square for the underlying block data.

Hmm, can't this simply re-use the square size as well?

Reply:
we actually need to use the OriginalSquare size here, otherwise the data root could be calculated differently.

in the future, we won't pass the square to the method, but for now we are